be/src/storage/lake/tablet_manager.cpp

```diff
@@ -243,7 +244,30 @@ StatusOr<TabletMetadataPtr> TabletManager::load_tablet_metadata(std::shared_ptr<
     auto t0 = butil::gettimeofday_us();
     auto metadata = std::make_shared<TabletMetadataPB>();
     ProtobufFile file(metadata_location, std::move(fs));
-    RETURN_IF_ERROR(file.load(metadata.get(), fill_cache));
+    auto s = file.load(metadata.get(), fill_cache);
```

Comment:
`segment` `delvec` files also have the same issue？

Reply:
currently only found the problem in meta, so deal with meta only now